### ISSUE TITLE
can not update PyPDF2==1.27.5

### ISSUE THREAD

**Issue by jwkaltz** [openoereb/pyramid_oereb#1568]
PyPDF2 can currently not be updated to a more recent version than 1.27.5, the automated tests fail with more recent versions.

The reason is not yet clear.

**Comment by vvmruder** [comment]
@jwkaltz Can this be closed?

**Comment by jwkaltz** [comment]
> @jwkaltz Can this be closed?

@vvmruder no, we are currently using an outdated version of PyPDF2 (but it's currently not a problem).

**Comment by mki-c2c** [comment]
fixed in https://github.com/openoereb/pyramid_oereb/pull/1597
